rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -2003,23 +1978,21 @@ static bool ppu_store_reservation(ppu_thread& ppu, u32 addr, u64 reg_value)
 		if (new_data == old_data)
 		{
 			ppu.last_faddr = 0;
-			return res.compare_and_swap_test(rtime, rtime + 128);
```

Comment:
It won't work for pure reservation notification where one waits on SPU LR and PPU starts dummy updating reservation with no value changes, keep this code under PPU accirate reservations.